apps/extension/src/screens/login.screen.tsx

```diff
@@ -15,10 +15,12 @@ import { isLoginRes, LoginResponse } from '@wtm/api';
 
 import clsx from 'clsx';
 import { updateIcon } from '../utils/updateIcon';
+import { apiClient } from '../utils/api.client';
 
-import { ROUTES } from '../hooks/use-extension-navigation';
+import { ExtensionRoutes } from '../hooks/use-extension-navigation';
 
 const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+const FRONTEND_URL = 'http://localhost:3000';
```

Comment:
@dcoscolla-intermedia I think you forgot to replace this hardoded value with the real front url that you intended to use